### DISCUSSION TITLE
General Praise & Project Roadmap

### DISCUSSION THREAD

**Discussion by lragnarsson** [uwdata/mosaic#269] [General]
I just found this project and it made me so excited! I just wanted to give my praise to the minds behind it, great job 👏 . It’s kind of the library I wanted ever since I started using duckdb, it just makes sense!

I spent the day integrating it in a data visualisation app, I have a typescript + react frontend with Mosaic + duckdb wasm. Fetching apache arrow tables from the backend (FastAPI in front of databricks). It’s quite a step up from the JavaScript objects and plotly that I had before. 

I am curious what your future plans and vision for the project is? What kind of features do you want before a v1.0 release? And how are you thinking about open source contributors?

Thanks 🙏

**Comment by domoritz** [comment]
Awesome to hear about your positive experience. We recently put a few project ideas and [tagged them as "projects"](https://github.com/uwdata/mosaic/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aprojects). These are not necessarily projects that are on the immediately roadmap but projects we think are interesting future research and we will suggest them to students we are working with. 

We welcome contributions but suggest creating an issue and getting feedback before starting a larger change. Also, see https://github.com/uwdata/mosaic/issues/408#issuecomment-2137731472 for ways to contribute.

**Comment by jheer** [comment]
Thanks for the kind words! 🙏

**Comment by mhkeller** [comment]
Adding another voice of praise for this project and really cool to see all of the recent activity! I've been working on incorporating it into an app I use for data analysis.
